lib/commands/slicemachine/commands/init.js

```diff
@@ -201,8 +216,15 @@ async function init() {
       const [protocol, base] = ctx.base.split('://');
       if (!protocol || !base) return [`Base url "${ctx.base}" is invalid: did you forget to specify protocol?`, null];
 
-      const apiEndpointUrl = `${protocol}://${r.domain}.${ctx.base.replace(/^(?:https?:\/\/)?(?:www\.)?/i, '')}/api/v2`;
-      return [null, _format(r.domain, r.magicUrl, apiEndpointUrl)];
+      const apiEndpoint = `${protocol}://${r.domain}.${ctx.base.replace(/^(?:https?:\/\/)?(?:www\.)?/i, '')}/api/v2`;
+      const configFile = Mustache.render(
+        readFileSync(path.join(__dirname, '../misc', 'templates', 'prismic.config.mustache'), 'utf8'),
+        {
+          apiEndpoint
+        }
+      )
+      writeFileSync('prismic.config.js', configFile, 'utf8');
```

Comment:
why to we need to add this file btw? isn't it nuxt plugin related? In this case it should be an instruction coming from the manifest more than an arbitrary code here

Reply:
It's still used by the sync method so I had to leave it there. But I did not want to make it part of the manifest as I'm willing to replace it with a simpler JSON configuration file (that is `sm.json` at the moment). The config file also contains the list of SM librairies used by the user and could be passed to prismic-nuxt.

Another solution would be to drop any configuration file but we would have to parse the user's config to at least find their endpoint. Also, I don't know yet how it would have to be handled in other frameworks